EnglishFilter/sections/specific.txt

```diff
@@ -254,6 +254,7 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 !
+usersdrive.com##a[onclick^="window.open("]
```

Comment:
This `usersdrive.com##a[onclick^="window.open("][rel^="noreferrer"]` is more accurate, please change it.

Reply:
I am not good in ads so yeah.